04-Multi-Band-Raster.Rmd

```diff
@@ -223,16 +220,29 @@ RGB_band2
 ```
 
 Notice that band 2 is band 2 of 3 bands.
+`band: 2  (of  3  bands)`
 
-`band        : 2  (of  3  bands)`
 
-#Raster Stacks in R
+#Challenge: Making sense of single band images
+If you compare the plots of band 1 (red) and band 2 (green) you can see subtle 
+differences in the image. The forest appears paler in band 2 (green). Why?  
+
 
-We have now explored several bands in a 3 band raster. Our data is a color image 
-which means it has atleast 3 bands. Let's bring in all of the bands and composite 
-it to create a final color RGB plot. To bring in all bands of a raster, we will use
-the `stack` function.
 
+Recall from our creation of the greyscale palette that greyscale is actually a 
```

Comment:
hmmmm... yea -- i think this could be a lot of things because also it has to do with image stretch - the range and distribution of colors in the image. Id probably remove this although it's a very good guess in terms of what is happening (the physics are correct). You could simply say the the green band (band 2) is brighter  / lighter in appearance as there are trees in the scene that are green. I'd probably leave it at that.